css/src/components/form/radio.scss

```diff
@@ -5,67 +5,33 @@ $radio-border-width: $control-border-width !default;
 $radio-dot-checked-color: $primary !default;
 $radio-dot-color: $text-subtle !default;
 $radio-dot-hover-unchecked-color: $control-border !default;
+$radio-dot-whitespace: 0.25em;
 $radio-timing-function: $input-timing-function !default;
 $radio-duration: $input-transition-duration !default;
 $radio-animation: boop !default;
 $radio-spacing: 0.5em !default;
 
-@mixin radio-dot {
-	position: absolute;
-	inset: 0;
-	width: $radio-dot-size;
-	height: $radio-dot-size;
-	margin: auto;
-	border-radius: $border-radius-rounded;
-	background-color: $body-background;
-	content: '';
-	overflow: hidden;
-}
-
-@mixin radio-dot-checked {
-	background: $radio-dot-checked-color;
-}
-
 .radio {
 	display: inline-flex;
-	position: relative;
-	align-items: flex-start;
 	line-height: 1.25;
 	cursor: pointer;
 
 	.radio-dot {
-		display: inline-block;
-		position: relative;
-		inset-block-start: 0.0625em;
-		flex-shrink: 0;
 		width: $radio-circle-size;
 		height: $radio-circle-size;
+		padding: $radio-dot-whitespace;
 		border: $radio-border-width solid $radio-border-color;
 		border-radius: $border-radius-rounded;
+		background-clip: content-box !important;
```

Comment:
For whatever reason, we're getting different results on Safari on iOS. (I've had issues with background clip on Safari and iOS chrome, so I knew to double check). Doesn't look to bad but unfortunately it's not consistent.

![image](https://user-images.githubusercontent.com/30843002/160658191-928ea7d6-8e39-4db3-a0b0-2f513fbf8ade.png)

Reply:
Thanks for letting me know. I'll close the PR for now until I find an alternative solution.